Downgrade minimum support version

vladyslav-iosdev opened the pull request:
Downgrade minimum support version
remove `[weak self]` in public func owner(domain: String, completion: @escaping StringResultConsumer ) {

`completion` get not called when using weak

merenkoff commented on UnstoppableDomainsResolution.podspec:
```diff
@@ -14,7 +14,7 @@ Pod::Spec.new do |spec|
   spec.social_media_url = 'https://twitter.com/unstoppableweb'
 
-  spec.ios.deployment_target = "12.1"
+  spec.ios.deployment_target = "10.0"
```
swift 5 not supported with iOS10 as I know.

merenkoff commented on UnstoppableDomainsResolution.podspec:
```diff
@@ -23,6 +23,6 @@ Pod::Spec.new do |spec|
   spec.resources = "Sources/UnstoppableDomainsResolution/Resources/**/*"
 
-  spec.dependency 'EthereumABI', '~> 1.3'
-  spec.dependency 'Base58Swift', '~> 2.1'
```
It is also not good point to remove exact versions from dependency

merenkoff commented on Sources/UnstoppableDomainsResolution/Resolution.swift:
```diff
@@ -136,13 +136,13 @@ public class Resolution {
     /// - Parameter  completion: A callback that resolves `Result`  with a `resolver address` for a specific domain or `Error`
     public func resolver(domain: String, completion: @escaping StringResultConsumer ) {
         let preparedDomain = prepare(domain: domain)
-        DispatchQueue.global(qos: .utility).async { [weak self] in
```
I will check this how it works in the profiler.

merenkoff commented on Resolution.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved:
```diff
@@ -24,26 +24,26 @@
-          "revision": "c5ee516c9616ec7dc2a000007ddb8bf5d9543c74",
-          "version": "0.13.1"
+          "revision": "3a2acbb32ab68215ee1596ee6004da8e90c3721b",
+          "version": "1.0.0"
```
good point to commit resolved versions

merenkoff commented:
If you need iOS 10 support I will check the possibility, how we could provide it...

@vladyslav-iosdev What project do you develop, where is our library is necessary?

merenkoff commented:
Also about weak. Please read our docs...

```NOTE: make sure an instance of the Resolution class is not deallocated until the asyncronous call brings in the result.```

vladyslav-iosdev commented:
> If you need iOS 10 support I will check the possibility, how we could provide it...

i am working on https://github.com/AlphaWallet/alpha-wallet-ios
i saw u commented pr (thanks a lot for u reply). I updated PR files,  removed all unnecessary changes.

vladyslav-iosdev commented:
@merenkoff i updated changes with ios 11, it should work with swift 5 version.

vladyslav-iosdev commented:
@merenkoff Thanks a lot ;)

merenkoff commented:
Congratulations! :tada: This was released as part of [Release 0.1.6](https://github.com/unstoppabledomains/resolution-swift/releases/tag/0.1.6) :rocket:<p align="right"><i>Generated by <a href="https://github.com/WeTransfer/GitBuddy">GitBuddy</a></i></p>

merenkoff commented:
Congratulations! :tada: This was released as part of [Release 1.1.5](https://github.com/unstoppabledomains/resolution-swift/releases/tag/1.1.5) :rocket:<p align="right"><i>Generated by <a href="https://github.com/WeTransfer/GitBuddy">GitBuddy</a></i></p>

merenkoff commented:
Congratulations! :tada: This was released as part of [Release 0.1.6](https://github.com/unstoppabledomains/resolution-swift/releases/tag/0.1.6) :rocket:<p align="right"><i>Generated by <a href="https://github.com/WeTransfer/GitBuddy">GitBuddy</a></i></p>

merenkoff commented:
Congratulations! :tada: This was released as part of [Release 0.1.7](https://github.com/unstoppabledomains/resolution-swift/releases/tag/0.1.7) :rocket:<p align="right"><i>Generated by <a href="https://github.com/WeTransfer/GitBuddy">GitBuddy</a></i></p>

merenkoff commented:
Congratulations! :tada: This was released as part of [Release 0.2.0](https://github.com/unstoppabledomains/resolution-swift/releases/tag/0.2.0) :rocket:<p align="right"><i>Generated by <a href="https://github.com/WeTransfer/GitBuddy">GitBuddy</a></i></p>
